whynotearth/BrowTricks
# Feature: Customer Home View

# Description

This PR addresses Issue #11.  We make a basic prototype of what the Customer Home view will look like. We hardcode some dummy data into the view as placeholders.

* `89d35ab`: We add one extension for height utilities in Tailwind config. We need this to set the height of the Menu Header (and later, Nav Headers)

* `ce12c47`: We move the 'Button' component to a sub-folder called 'ui' where we will put more common components used across both Tenant and Customer views

* `dae0b0e`: We add the basic MenuHeader component which has the hamburger menu icon at the top left. We will use this component for the following purposes in the Customer views, as seen in the Figma designs:

![image](https://user-images.githubusercontent.com/20192983/80413908-0be7f400-8885-11ea-987b-166680722702.png)

* `b21fbd8`: We add the first iteration of the Customer Home View component.

* `3ff331b`: We add placeholder routes to access the Customer Home View component from `/`

# How to test

Go to `/` or `/shop/test-tenant-slug` and see the result

![image](https://user-images.githubusercontent.com/20192983/80414132-65e8b980-8885-11ea-8cc6-97d2d95ec1a2.png)
